src/components/dev-hub/project-sign-up-form.js

```diff
@@ -73,20 +73,37 @@ const Form = React.memo(({ setSuccess, success }) => {
             </ErrorMessage>
             <StyledInput
                 value={name}
+                maxLength="50"
                 required
                 placeholder="Name"
+                pattern="^[A-Za-zÀ-ÿ ,.'-]+$"
                 onChange={e => setName(e.target.value)}
+                onInput={e => e.target.setCustomValidity('')}
+                onInvalid={e =>
+                    e.target.setCustomValidity(
+                        'Names should only contain letters. e.g. John Doe'
+                    )
+                }
             />
             <StyledInput
+                type="email"
                 value={email}
                 required
                 placeholder="Email Address"
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,3}$"
```

Comment:
the `email` type also gives us some validation checks for free, so I don't think we need this pattern or `onInput` or `onInvalid` here. Were you seeing something else?

Reply:
The `email` type validation does not seem to check for the `.com` part. For example, `example@email` is considered valid.

Reply:
aha! Nice find!